docs/flocks-settings/webhooks.md

```diff
@@ -262,7 +262,7 @@ endpoints:
         description: A valid Slack webhook 
     response: A JSON structure with the result indicator.
   splunk_get:
-    name: Show Splunk Webhook on flock
+    name: Show Splunk Webhook on the flock
```

Comment:
Could we instead make this "Show Splunk Webhook for a Flock"

Reply:
done 🔨